web/src/settings_streams.ts

```diff
@@ -1,4 +1,5 @@
 import $ from "jquery";
+import type {Instance} from "tippy.js";
```

Comment:
Let’s use namespace imports for Tippy going forward.

- #30265

```suggestion
import * as tippy from "tippy.js";
```

Reply:
Sure. It did get confusing sometimes, reading the code and wondering what `Instance` is this referring to. Using namespace would improve readability for sure. I went with the following change as it would only import declarations (which is what we want here).
```suggestion
import type * as tippy from "tippy.js";
```